linux/Jenkinsfile

```diff
@@ -206,7 +206,9 @@ def uploadRPMArtifacts(distro) {
             'rpm/fedora/36',
             'rpm/oraclelinux/8',
             'rpm/amazonlinux/2',
-            'rpm/oraclelinux/7'
+            'rpm/oraclelinux/7',
+            'rpm/almalinux/8',
```

Comment:
1. AlmaLinux 8/9 does not support `armv7hl` defined in archs.
https://github.com/adoptium/installer/blob/master/linux/Jenkinsfile#L218-L224

2. AlmaLinux 8 does not support `s390x`.

Reply:
1. The Oracle Linux 8, Amazon Linux 2, Rocky Linux 8 only supports `x86_64` and `AArch64` in the list of the supported OSes.

2. As said in #499, AlmaLinux OS 9 has `x86_64`, `AArch64`, `ppc64le` and `s390x` support. The `s390x` support on the way for the AlmaLinux OS 8.